packages/types/index.ts

```diff
@@ -3,6 +3,7 @@ export { default as Package } from "./package";
 export { default as Fill } from "./fill";
 export { default as Frontity } from "./frontity";
 export { Action, AsyncAction, ServerAction, Context } from "./action";
+export { Server, AsyncServer, FrontityContext } from "./server";
```

Comment:
What's the difference between `FrontityContext`, and action's `Context`? Can't we get rid of `Context` and rename `FrontityContext` to `Context` (and use it in `ServerActions` as well)?

Reply:
Yes, I think it is safe to get rid of `Context`.